embed-map-single-year.js

```diff
@@ -9,41 +9,47 @@ var stylesByLayer = {
 };
 
 addEventListener('load', function () {
-  var params = new URLSearchParams(location.hash.substr(1));
-  var style = stylesByLayer[params.get('layer') || ''] || stylesByLayer.O;
-  
-  upgradeLegacyHash();
-  
+  let
+    bbox = JSON.parse(`[${new URLSearchParams(location.search).get('bbox')}]`),
+    params = new URLSearchParams(location.hash.substring(1)),
+    style = stylesByLayer[params.get('layer') || ''] || stylesByLayer.O
+  ;
+
   window.map = new maplibregl.Map({
     container: 'map',
-    hash: 'map',
+    hash: true,
```

Comment:
When this is `true`, MapLibre turns the hash into `#zoom/latitude/longitude` as you pan and zoom around. This format is incompatible with the code in the `hashchange` handler below. Can we move the `bbox` parameter into the hash instead of using it as a standard query parameter?